include/divideconquer.h

```diff
@@ -20,7 +20,13 @@
 #ifndef GRPPI_DIVIDECONQUER_H
 #define GRPPI_DIVIDECONQUER_H
 
-#include "common/common.h"
+#include "common/sequential_execution.h"
```

Comment:
This includes are not needed here.

Put the includes in the places they are needed.